x-pack/plugins/observability/public/components/alerts_flyout_body.tsx

```diff
@@ -42,6 +42,14 @@ interface FlyoutProps {
   id?: string;
 }
 
+// For APM Latency threshold rule, threshold is in ms but the duration formatter works with microseconds
```

Comment:
> For APM Latency threshold rule, threshold is in ms but the duration formatter works with microseconds

IMO, this is a little bit confusing. Is there a way where we can have a consistent and unified unit for apm.transaction_duration ?

Reply:
I agree, and I created a [ticket](https://github.com/elastic/kibana/issues/156255) for it. Yesterday, I discussed this issue with my team, but we haven't decided how to fix it, and I want to make sure this issue is fixed for `v8.8`.